src/octoprint/filemanager/analysis.py

```diff
@@ -262,7 +260,7 @@ def throttle():
 			self._gcode.load(self._current.absolute_path, self._current.printer_profile, throttle=throttle_callback)
 	
 			result = dict()
-			result["warning"]=self._gcode.getWarning()
+			result["printingArea"]={"minX" : self._gcode.minX, "minY" : self._gcode.minY, "minZ" : self._gcode.minZ, "maxX" : self._gcode.maxX, "maxY" : self._gcode.maxY, "maxZ" : self._gcode.maxZ}
```

Comment:
Missing white space in assignment, and shouldn't all be on one line - line breaks and white space in general are a GOOD thing to have in code ;)